src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -69,6 +69,8 @@
 _USER_ASSIGNED_IDENTITY = 'userAssignedIdentity'
 _ASSIGNED_IDENTITY_INFO = 'assignedIdentityInfo'
 
+AZ_LOGIN_MESSAGE = "Please run 'az login' to setup account."
```

Comment:
My only ask would be to prefix this with underscore `_AZ_LOGIN_MESSAGE`. We are trying to be conscious of expanding the public interface for core components.